specs/mediawiki/v1/content.yaml

```diff
@@ -422,6 +454,22 @@ paths:
             $ref: '#/definitions/problem'
       x-backend-request:
         uri: /{domain}/sys/parsoid/data-parsoid/{title}
+      x-amples:
+        - title: Should get a latest data-parsoid of a page by title
+          request:
+            params:
+              title: Foobar
+            headers:
+              cache-control: no-cache
```

Comment:
No need for this header, the previous test for `html/{title}` already fetches the newest page bundle from Parsoid